File: __tests__/fixtures/output/python/python3/nested.py
import http.client

conn = http.client.HTTPSConnection("httpbin.org")

conn.request("GET", "/anything?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value")

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: src/targets/ocaml/cohttp/fixtures/nested.ml
open Cohttp_lwt_unix
open Cohttp
open Lwt

let uri = Uri.of_string "https://httpbin.org/anything?foo%5Bbar%5D=baz%2Czap&fiz=buz&key=value" in

Client.call `GET uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)